sql/upgrade/sl3.0.sql

```diff
@@ -896,19 +920,24 @@ UPDATE business_unit_class
  WHERE id = 2
    AND EXISTS (select 1 from :slschema.project);
 
+ALTER TABLE gl DISABLE TRIGGER gl_audit_trail;
+ALTER TABLE gl DISABLE TRIGGER gl_prevent_closed;
```

Comment:
Do you need to disable this trigger? I'd think you didn't set up the year-end table yet (and if you did, maybe it's an option to delay that), so that you don't need to disable the trigger... Or is this a performance measure? If the latter, could you please add a comment to that effect, substantiating the choice with your experience in your testing (e.g. by "disabling this trigger I was able to migrate a 2mio transactions company in 1min instead of 15mins")?

Reply:
Removed